pkg/app/appdisc/discovery_manager.go

```diff
@@ -0,0 +1,50 @@
+package appdisc
```

Comment:
Should the file be named discovery_updater.go again?

Reply:
oh! I renamed everything, and forgot about file name.
Done now.